packages/core/admin/server/services/api-token.js

```diff
@@ -1,30 +1,106 @@
 'use strict';
 
 const crypto = require('crypto');
+const { omit, difference, isEmpty, map, isArray } = require('lodash/fp');
+const { ValidationError, NotFoundError } = require('@strapi/utils').errors;
+const constants = require('./constants');
 
 /**
- * @typedef {'read-only'|'full-access'} TokenType
+ * @typedef {'read-only'|'full-access'|'custom'} TokenType
```

Comment:
We should move all the types defined in jsdoc comments to .ts files before releasing if possible

Reply:
Let's iterate on this for 4.4.1